tests/test_terrain.py

```
@@ -0,0 +1,122 @@
+"""
+Tests for the terrain generation system.
+"""
+
+import unittest
+import numpy as np
```

Ruff (0.8.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)